### PR TITLE
use `app:update` instead of deprecated `rails:update` [ci skip]

### PR DESCRIPTION
**Pull request by y-yagi**
`rails:update` was deprecated in 6fb31638c8b61731103d4963272755b217a2df87

### CODE REVIEW

**Comment by arunagw** [comment]
Thanks :-)

**Comment by y-yagi** [comment]
:smile:

File: guides/source/upgrading_ruby_on_rails.md [hunk 1]
```diff
@@ -44,13 +44,13 @@ TIP: Ruby 1.8.7 p248 and p249 have marshaling bugs that crash Rails. Ruby Enterp
 
 ### The Task
 
-Rails provides the `rails:update` rake task. After updating the Rails version
+Rails provides the `app:update` rake task. After updating the Rails version
```
Comment (matthewd) [inline guides/source/upgrading_ruby_on_rails.md]: Not a rake task.

(I mean.. it is, but that's now an implementation detail: if you don't run it with rake, then as far as the user's concerned, it's not a rake task.)

Comment (y-yagi) [inline guides/source/upgrading_ruby_on_rails.md]: I see.
I will create PR.

Comment (prathamesh-sonpatki) [inline guides/source/upgrading_ruby_on_rails.md]: Fixed by https://github.com/rails/rails/pull/23943

**Pull request merged**